source/features/more-dropdown-links.tsx

```diff
@@ -28,16 +30,14 @@ export async function unhideOverflowDropdown(): Promise<boolean> {
 	return true;
 }
 
-async function init(): Promise<void> {
+async function addDropdownItems(repoNavigationDropdown: HTMLElement): Promise<void> {
 	const reference = getCurrentGitRef() ?? await getDefaultBranch();
 	const compareUrl = buildRepoURL('compare', reference);
 	const commitsUrl = buildRepoURL('commits', reference);
 	const branchesUrl = buildRepoURL('branches');
 	const dependenciesUrl = buildRepoURL('network/dependencies');
 	await unhideOverflowDropdown();
```

Comment:
If it's hidden, the observer will never fire. The feature is probably only working because `clean-repo-tabs` is enabled. 

I think the fix is to move this before the observer. Please. Verify the bug by disabling the other feature